examples/default.yaml

```diff
@@ -229,13 +229,21 @@ firmware:
   legacyBIOS: null
 
 video:
-  # QEMU display, e.g., "none", "cocoa", "sdl", "gtk", "default".
+  # QEMU display, e.g., "none", "cocoa", "sdl", "gtk", "vnc", "default".
   # Choosing "none" will hide the video output, and not show any window.
+  # Choosing "vnc" will use a network server, and not show any window.
   # Choosing "default" will pick the first available of: gtk, sdl, cocoa.
-  # As of QEMU v6.2, enabling this is known to have negative impact
+  # As of QEMU v6.2, enabling anything but none or vnc is known to have negative impact
   # on performance on macOS hosts: https://gitlab.com/qemu-project/qemu/-/issues/334
   # 🟢 Builtin default: "none"
   display: null
+  # VNC (Virtual Network Computing) is a platform-independent graphical
+  # desktop-sharing system that uses the Remote Frame Buffer protocol (RFB)
+  vnc:
+    # VNC display, e.g.,"to=L", "host:d", "unix:path", "none"
```

Comment:
What does "to=L" mean?
And how is "none" expected to be used?

Reply:
This was from the upstream documentation, could copy the detailed explanations

Reply:
to=L
               With this option, QEMU will try next available VNC displays,
               until the number L, if the origianlly defined "-vnc display"
               is not available, e.g. port 5900+display is already used by
               another application. By default, to=0.

           host:d
               TCP connections will only be allowed from host on display d.
               By convention the TCP port is 5900+d. Optionally, host can be
               omitted in which case the server will accept connections from
               any host.

           unix:path
               Connections will be allowed over UNIX domain sockets where
               path is the location of a unix socket to listen for
               connections on.

           none
               VNC is initialized but not started. The monitor "change"
               command can be used to later start the VNC server.